lib/KettleConfigLoader.js

```diff
@@ -146,19 +147,19 @@ kettle.config.createDefaultsImpl = function (prefix, filePath) {
 /** Convert a Kettle "config" as specified by a base path and configName value into a set of component defaults
  * constituting a runnable Kettle application. This will recursively load as grades any "loadConfigs" and "mergeConfigs" specified in the root
  * config file, for mergeConfigs, merge their configuration into the root config.
- * @param options {Object} Contains fields:
+ * @param {Object} options - Contains fields:
  *     configName {String} a value that will be looked up to a "config" file in the supplied
  * directory by appending the <code>.json</code> suffix
  *     configPath {String} A fully qualified directory name holding config files, or one prefixed with a %module base path reference
  * @return {String} the global name of a Fluid component which can be invoked to instantiate the Kettle application
```

Comment:
For consistency with similar docs, probably "the" should be capitalised.

Reply:
Should now be normalised throughout.